### DISCUSSION TITLE
Outreachy Applicant Contribution

### DISCUSSION THREAD

**Discussion by Codde24** [LesFruitsDefendus/saskatoon-ng#229] [Outreachy]
@tristanlatr 
I have worked on something and ready to contribute, how do i move forward with my contribution here

**Comment by tristanlatr** [comment]
Hi @Codde24, 

First step is to install Saskatoon in your local computer (refer to `INSTALL.md`), then create a branch for your issue/feature, then you can start hacking. When you feel like you reached a stable point, commit your changes and open a Pull Request. 

I hope I answered your question, don't hesitate to ask more precise follow-up questions,

Talk to you later,

**Reply by Codde24** [reply]
thank you, question has been answered
